frontend/src/lib/components/PayGateMini/payGateMiniLogic.tsx

```diff
@@ -36,11 +36,14 @@ export const payGateMiniLogic = kea<payGateMiniLogicType>([
         productWithFeature: [
             (s) => [s.billing],
             (billing) => {
-                let foundProduct: BillingProductV2Type | BillingProductV2AddonType | undefined =
-                    billing?.products?.find((product) => product.features?.some((f) => f.key === props.featureKey))
+                const allAddons = billing?.products?.map((product) => product.addons).flat() || []
```

Comment:
```suggestion
                // check addons first since their features are rolled up into the parent
                const allAddons = billing?.products?.map((product) => product.addons).flat() || []
```

I feel like because the order matters here but it's not super apparent as to why, a comment is worthwhile

Reply:
Good call 👍